sled-agent/src/services.rs

```diff
@@ -1919,30 +1919,49 @@ impl ServiceManager {
         {
             return Err(Error::BadServiceRequest {
                 service: name,
-                message: "Should initialize zone twice".to_string(),
+                message: "Should not initialize zone twice".to_string(),
             });
         }
 
+        // We initialize all the zones we can, but only return the first error.
         let local_existing_zones = Arc::new(Mutex::new(existing_zones));
+        let first_err = Arc::new(Mutex::new(None));
```

Comment:
Not a blocker, but as written shouldn't this be `last_err`?

Reply:
Done in https://github.com/oxidecomputer/omicron/pull/3709 , pulled it out to not block the build for folks MUPdating